src/aws/lambda.tf

```diff
@@ -131,7 +131,7 @@ resource "aws_lambda_function" "marsha_migrate_lambda" {
   environment {
     variables = {
       S3_SOURCE_BUCKET        = aws_s3_bucket.marsha_source.id
-      LAMBDA_ENCODE_NAME      = aws_lambda_function.marsha_encode_lambda.function_name
+      LAMBDA_CONVERT_NAME     = aws_lambda_function.marsha_convert_lambda.function_name
```

Comment:
This is an environment variable used by the lambda you are modifying. This means you are breaking the migrate lambda if you don't also change the variable name in its code.

Reply:
Yes, terraform apply is broken due to this change.
I'm fixing it right now.